content/posts/2022/reana-client-appimage.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Standalone reana-client application executables for Linux systems"
+date: 2022-06-01T08:00:00+01:00
+tags:
+  - client
+  - linux
+---
+
+The usual way of creating, executing and managing REANA workflows is by means
+of a [reana-client](https://docs.reana.io/getting-started/first-example/)
+command-line tool. However, installing `reana-client` can be painful in certain
+situations, for example when it conflicts with your other Python project
+dependencies and when you don't want to be switching Python virtual
+environments all the time. In these cases, using a fully standalone
+`reana-client` executable would be desirable.
+
+<!--more-->
+
+As of REANA 0.8.0, we have started publishing standalone `reana-client`
+executables for Linux operating systems using the
+[AppImage](https://appimage.org/) technology. The published application
+executables contain `reana-client` command-line tool together with Python and
+all the necessary dependent libraries embedded in the bundle, so that the
+clients run fully independently of your local environment.
+
+You can download `reana-client` standalone executables from our [GitHub release
+pages](https://github.com/reanahub/reana-client/releases), for example:
+
+```console
+$ wget https://github.com/reanahub/reana-client/releases/download/0.8.1/reana-client-0.8.1-x86_64.AppImage
+```
+
+If you place the executable into some convenient directory found in your
+`PATH`, such as `$HOME/.local/bin`:
+
+```console
+$ chmod u+x ./reana-client-0.8.1-x86_64.AppImage
+$ mv ./reana-client-0.8.1-x86_64.AppImage $HOME/.local/bin/reana-client-0.8.1
+```
+
+you will be able to use `reana-client-0.8.1` executable from anywhere:
+
+```console
+$ cd myanalysis
+$ reana-client-0.8.1 ping
+REANA server: https://reana.cern.ch
+REANA server version: 0.9.0a5
+REANA client version: 0.8.1
+Authenticated as: John Doe <john.doe@example.org>
+Status: Connected
+```
+
+Please note that the `reana-client` AppImage executable format works only on
+Linux operating systems. It should be supported by all major Linux
+distributions. Please let us know if encounter any troubles on your favourite
+Linux distribution.
+
+If you are using another operating system, such as macOS, you will not be able
+to use these images. But don't worry! We are considering creating standalone
+statically-linked `reana-client` executables using Golang in the near future.
+This will provide the same functionality (and be much faster as a bonus). Stay
+tuned for more news later in the year.
```

Comment:
```suggestion
to use these images. But don't worry! We are considering switching to Go as the programming language
for `reana-client` in the near future, which will let us easily create standalone cross-platform executables.
This new version of `reana-client` will provide the same functionalities as the old one, while also being much faster as a bonus. Stay
tuned for more news later in the year.
```
WDYT?

Reply:
Hmm, "switching" would send too strong a message, e.g. we may want to retain Python APIs for RECAST and notebooks etc.  I have reformulated tho paragraph into:

```
If you are using another operating system, such as macOS, you will not be able
to use these executables. But don't worry! We are considering creating
statically-linked cross-platform `reana-client` executables using Go
programming language in the near future. Stay tuned for more news later in the
year.
```